apps/client/sttr_build/parser.js

```diff
@@ -52,165 +61,233 @@ const getAutofillResolverKey = (questionText) => {
   }
 };
 
+const filt = (arr, tagName) => arr.filter(({ name }) => name === tagName);
```

Comment:
woooow this file... i need to take some time for this 🕐

Reply:
Yeah, xml parsing 🥳 
Will add some comments here and there, especially the util's.